lib/tapioca/commands/annotations.rb

```diff
@@ -195,7 +195,7 @@ def add_header(name, content)
 
         contents = content.split("\n")
```

Comment:
As per my other comment:
```suggestion
        # Split contents into newlines and ensure trailing empty lines are included
        contents = content.split("\n", -1) 
```